portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -1,18 +1,20 @@
 package com.google.sps.servlets;
-
+                                
```

Comment:
Very minor since we don't have an automatic formatter, but no need to have these spaces in the blank lines.